ivadomed/loader/loader.py

```diff
@@ -53,6 +41,8 @@ def load_dataset(bids_df, data_list, transforms_params, model_params, target_suf
     Note: For more details on the parameters transform_params, target_suffix, roi_params, contrast_params,
     slice_filter_params and object_detection_params see :doc:`configuration_file`.
     """
+    from ivadomed.loader.bids3d_dataset import Bids3DDataset
+    from ivadomed.loader.bids_dataset import BidsDataset
```

Comment:
Hmm... Great spot! I thought this was causing circular import but it apparently doesn't because base on the latest check. Moving it to the top then in this case.

Typically, import is done at module level such that it is only importing it once and won't have to do it at functional level. 

Sometimes, structural/architecture reason arises, that if module functionalities are not properly segregated and have dependencies on each other, it will literally throw a partial module initialized error. I was mistaken in this case so we should move it to the top. Corrected.